views/engineering_garage/edit_teammates.erb

```diff
@@ -0,0 +1,67 @@
+<h1 class="dcf-txt-h3" id="Edit_Teammates">Edit Teammates</h1>
```

Comment:
I feel like this could get put into that edit project page. That table is basically the same except for the delete button and the user input